docs/getting_started.md

```diff
@@ -190,25 +190,48 @@ In a similar way create a stage **Prod (AKS** and add a single task to the job *
 
 Specify task parameters as it is shown in the table below:
 
-| Parameter | Value |
-| --- | --- |
-| Display Name | Azure ML Model Deploy |
-| Azure ML Workspace | mlops-AML-WS |
-| Inference config Path | `$(System.DefaultWorkingDirectory)/_ci-build/mlops-pipelines/code/scoring/inference_config.yml` |
-| Model Deployment Target | Azure Kubernetes Service |
-| Select AKS Cluster for Deployment | YOUR_DEPLOYMENT_K8S_CLUSTER |
-| Deployment Name | mlopspython-aks |
-| Deployment Configuration file | `$(System.DefaultWorkingDirectory)/_ci-build/mlops-pipelines/code/scoring/deployment_config_aks.yml` |
-| Overwrite existing deployment | X |
+| Parameter                         | Value                                                                                                |
+| --------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| Display Name                      | Azure ML Model Deploy                                                                                |
+| Azure ML Workspace                | mlops-AML-WS                                                                                         |
+| Inference config Path             | `$(System.DefaultWorkingDirectory)/_ci-build/mlops-pipelines/code/scoring/inference_config.yml`      |
+| Model Deployment Target           | Azure Kubernetes Service                                                                             |
+| Select AKS Cluster for Deployment | YOUR_DEPLOYMENT_K8S_CLUSTER                                                                          |
+| Deployment Name                   | mlopspython-aks                                                                                      |
+| Deployment Configuration file     | `$(System.DefaultWorkingDirectory)/_ci-build/mlops-pipelines/code/scoring/deployment_config_aks.yml` |
+| Overwrite existing deployment     | X                                                                                                    |
 
 **Note:** Creating of a Kubernetes cluster on AKS is out of scope of this tutorial, so you should take care of it on your own.
 
-Save the pipeline and craete a release to trigger it manually. Once the pipeline exection is finished, check out deployments in the **mlops-AML-WS** workspace.
+**Deploy trained model to Azure Web App for containers**

```

Comment:
It's not clear why we should do that. Perhaps it is worth mentioning, that it is an optional step if you don't work with ACI or AKS and prefer Web App